csi.proto

```diff
@@ -502,6 +529,179 @@ message ControllerServiceCapability {
     RPC rpc = 1;
   }
 }
+message CreateSnapshotRequest {
```

Comment:
source to create the snapshot (i.e. volume_id) missing here?

Reply:
Oops:).  Good catch!  "volume_id" was in the draft design doc, but it looks like I removed it here by accident when removing "version".  Thanks!

Reply:
Fixed.